packages/ui/src/types/authMachine.ts

```diff
@@ -68,15 +68,15 @@ export type InvokeActorEventTypes =
   | 'done.invoke.resetPasswordActor';
 
 export type AuthEventTypes =
-  | 'SIGN_IN'
-  | 'SIGN_UP'
-  | 'SIGN_OUT'
-  | 'SUBMIT'
-  | 'RESEND'
   | 'CHANGE'
   | 'FEDERATED_SIGN_IN'
+  | 'RESEND'
   | 'RESET_PASSWORD'
+  | 'SIGN_IN'
+  | 'SIGN_OUT'
+  | 'SIGN_UP'
   | 'SKIP'
+  | 'SUBMIT'
```

Comment:
I love when PRs include alphabetizing things. <3

Reply:
I actually used the extension you suggested! Sorting feels good ✨